simple_network_sim/common.py

```diff
@@ -13,7 +14,15 @@
 DEFAULT_GITHUB_REPO = "https://github.com/ScottishCovidResponse/simple_network_sim.git"
 
 
-# CurrentlyInUse
+class IssueSeverity(Enum):
+    """
+    This class defines the severity levels for issues found while running the model or loading data.
+    """
+    LOW = 1
+    MEDIUM = 5
+    HIGH = 10
```

Comment:
I'm fine with these number, but just curious: are they from convention for this sort of thing, or did we just have to pick some?

Reply:
I've made them up. As far as I know there aren't any agreed upon convention for these values.